### PR TITLE
A constant map is used to pass constants

### PR DESCRIPTION
**Pull request by LiZhenCheng9527**
**What type of PR is this?**

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:
A part of  #977 

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

File: bpf/include/bpf_log.h [hunk 1]
```diff
@@ -47,7 +47,7 @@ struct {
     __uint(max_entries, 1);
     __uint(key_size, sizeof(__u32));
     __uint(value_size, sizeof(__u32));
-} bpf_log_level SEC(".maps");
+} constant_map SEC(".maps");
```
Comment (hzxuzhonghu) [inline bpf/include/bpf_log.h:50]: This is a worse name without concrete meaning

**Comment by LiZhenCheng9527** [comment]
#977

File: pkg/bpf/ads/loader.go [hunk 2]
```diff
@@ -75,7 +75,7 @@ func (sc *BpfAds) Start() error {
 }
 
 func (sc *BpfAds) GetBpfLogLevelMap() *ebpf.Map {
-	return sc.SockConn.BpfLogLevel
+	return sc.SockConn.BpfConfigMap
```
Comment (hzxuzhonghu) [inline pkg/bpf/ads/loader.go:78]: There is a nother file loader_enahanced.go need to be updated

File: bpf/include/common.h [hunk 3]
```diff
@@ -131,6 +131,13 @@ struct {
     __type(value, struct buf);
 } tmp_buf SEC(".maps");
 
+struct {
+    __uint(type, BPF_MAP_TYPE_ARRAY);
+    __uint(max_entries, 1);
+    __uint(key_size, sizeof(__u32));
+    __uint(value_size, sizeof(__u32));
+} bpf_config_map SEC(".maps");
```
Comment (hzxuzhonghu) [inline bpf/include/common.h:139]: Last comment, `kmesh_config_map`

Comment (LiZhenCheng9527) [reply bpf/include/common.h]: Done

**Review by hzxuzhonghu — APPROVED** [review]
/lgtm

**Pull request merged**